modules/svg/data.js

```diff
@@ -161,6 +167,17 @@ export function svgData(projection, context, dispatch) {
     }
 
 
+    function containsGeometryCollection(gj) {
+        var features = getFeatures(gj);
+
+        var geomCollections = features.filter(function(feature) {
+            return feature.geometry.type === 'GeometryCollection';
+        });
```

Comment:
I think this could be made more performant by using `.some()` so that it can stop looping at the first GeometryCollection.